src/calculateRentalCost.js

```diff
@@ -4,7 +4,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const rentCost = 40;
+
+  if (days >= 3 && days < 7) {
+    return rentCost * days - 20;
+  }
+
+  if (days >= 7) {
+    return rentCost * days - 50;
+  }
+
+  return rentCost * days;
```

Comment:
The function name 'calculateRentalCost' is descriptive and clear. However, the hard-coded literals for the number of days and discount amounts should be replaced with appropriately named constants to improve readability and maintainability of the code.